pwiz_tools/Skyline/Model/IonMobility/IonMobilityLibraryManager.cs

```diff
@@ -124,9 +125,11 @@ private IonMobilityLibrary LoadIonMobilityLibrary(IDocumentContainer container,
         private static IonMobilityLibrary GetIonMobilityLibrary(SrmDocument document)
         {
             if (document == null)

```

Comment:
I would have probably done:
            return document?.Settings?.TransitionSettings?.IonMobilityFiltering?.IonMobilityLibrary 
                   ?? IonMobilityLibrary.NONE;

Reply:
That is tidy. I'll make the change and rerun tests before merging.